ui_tests/src/test/java/com/saucedemo/pageObjects/SignInPage.java

```diff
@@ -37,7 +37,7 @@ public void setEmail(String username) {
     }
 
     @And("^I set '(.*)' as password on Sign in page$")
-    public void setPassword(String password) {
+    public void problem_usersetPassword(String password) {
```

Comment:
@OzodaAripova Please undo this change (the method name shouldn't be changed from setPassword to  problem_usersetPassword)

Reply:
@OzodaAripova ^^

Reply:
@OzodaAripova ^^